docs/en/configuration.rst

```diff
@@ -512,3 +512,17 @@ Within the bootstrap script, the following variables will be available:
      * @var \Symfony\Component\Console\Output\OutputInterface $output The executing command's output object
      * @var \Phinx\Console\Command\AbstractCommand $context the executing command object
      */
+
+Feature Flags
+-------------
+
+For some breaking changes, Phinx offers a way to opt-out of new behavior. The following flags are available:
+
+* ``unsigned_pks``: Should Phinx create primary keys as unsigned integers? (default: ``true``)
+* ``column_null``: Should Phinx create columns as null by default? (default: ``true``)
```

Comment:
Are we usually using `snake_case` or `camelBacked` style for config options?

Reply:
`snake_case`, e.g.:

* `version_order`
* `default_environment`
* database config options (for example `attr_case` or `mysql_attr_ignore_space`)

etc.

Reply:
I also made this mistake thinking we used camelBack. Agree using the same.